app/models/user.rb

```diff
@@ -19,6 +19,14 @@ def self.find_by_slack_mention(user_name)
     User.where(user_name =~ /^<@(.*)>$/ ? { user_id: Regexp.last_match[1] } : { user_name: user_name }).first
   end
 
+  def self.find_many_by_slack_mention(user_names)
```

Comment:
Since `find_by_slack_mention` doesn't raise an exception when a user isn't found, `find_many_by_slack_mention` shouldn't either. You pick: either change `find_by_slack_mention` to `find_by_slack_mention!` (bang method) and fail in there (and adjust callers) or return an array where some users may be `nil` and handle it in the caller. I don't have a preference.